en/micro-integrator/docs/develop/deploy-artifacts.md

```diff
@@ -14,6 +14,12 @@ Download and set up a Micro Integrator server in your VM and deploy the composit
 
 See the instructions in [using a remote Micro Integrator](../using-remote-micro-integrator).
 
+## Deploy artifacts in an ESB Profile
```

Comment:
need to change this as remote Micro Integrator. ( we do mention about ESB profile in MI docs, as Mi has only ESB profile - its applicable in EI 6x space only )